src/main/java/com/google/sps/CloudVisionAPI.java

```diff
@@ -0,0 +1,23 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+/**
+ * Interface used for Cloud Vision API related functionality.
+ */
+public interface CloudVisionAPI {
+  protected BatchAnnotateImagesResponse querier(List<AnnotateImageRequest> requests)
```

Comment:
Do we need to have "List<AnnotateImageRequest> requests"? Wouldn't you send only one image?

Reply:
The [documentation](https://googleapis.dev/java/google-cloud-vision/1.12.0/com/google/cloud/vision/v1/ImageAnnotatorClient.html) says it only takes `List<AnnotateImageRequest>`.
I saw that `batchAnnotateImagesCallable()` is used for multiple images.